python/tvm/meta_schedule/testing/relay_workload.py

```diff
@@ -86,27 +89,29 @@ def _get_network(
         input_name = "input0"
         shape_list = [(input_name, input_shape)]
         mod, params = relay.frontend.from_pytorch(scripted_model, shape_list)
+        passes = [relay.transform.RemoveUnusedFunctions()]
+        if layout == "NHWC":
```

Comment:
If layout is `NCHW` do we need to also add layout transformation?

Reply:
added a comment saying pytorch is imported as nchw